apps/platform/trpc/routers/userRouter/securityRouter.ts

```diff
@@ -1081,5 +1115,226 @@ export const securityRouter = router({
       });
 
       return { success: true };
-    })
+    }),
+  deleteAccountPre: elevatedProcedure.query(async ({ ctx }) => {
+    const { db, account } = ctx;
+    const accountOrgsQuery = await db.query.accounts.findFirst({
+      where: eq(accounts.id, account.id),
+      columns: {
+        username: true
+      },
+      with: {
+        orgMemberships: {
+          columns: {
+            id: true,
+            role: true
+          },
+          with: {
+            org: {
+              columns: {
+                publicId: true,
+                name: true,
+                avatarTimestamp: true,
+                ownerId: true,
+                shortcode: true
+              }
+            }
+          }
+        }
+      }
+    });
+
+    if (!accountOrgsQuery) {
+      throw new TRPCError({
+        code: 'NOT_FOUND',
+        message: 'Account not in any orgs'
```

Comment:
its more of `account not found`, but this error would never be thrown as account is already defined in a account procedure.